### DISCUSSION TITLE
Can't install neorg?

### DISCUSSION THREAD

**Discussion by Bogfoot** [nvim-neorg/neorg#783] [Q&A]
Keep getting this error message:
```
Failed to run `config` for neorg
...local/share/nvim/lazy/lazy.nvim/lua/lazy/core/loader.lua:330 attempt to index a boolean value.
```
I'm using Neovim 0.8.3 and lazy nvim plugin manager, also tired on latest version as of toady. Don't really know what to make of this.

**Comment by andreadev-it** [comment]
Hi Bogfoot, could you please post your neorg configuration for lazy.nvim?

**Reply by Bogfoot** [reply]
Hello andreadev-it, I just copy-pasted the configuration from the repo and added an extra "work" folder to "dirman" workspaces.
```
{
        "nvim-neorg/neorg",
        build = ":Neorg sync-parsers",
        opts = {
            load = {
                ["core.defaults"] = {}, -- Loads default behaviour
                ["core.norg.concealer"] = {}, -- Adds pretty icons to your documents
                ["core.norg.dirman"] = { -- Manages Neorg workspaces
                    config = {
                        workspaces = {
                            notes = "~/Notes/PhD",
                            work = "~/Notes/Work",
                        },
                    },
                },
            },
        },
        dependencies = { { "nvim-lua/plenary.nvim" } },
    }
```

**Reply by Bogfoot** [reply]
Ok so I fixed the issue, but to be completely honest I have no clue how. While testing something else I removed a line in my lsp configuration files regarding lua, exited neovim, back into the same file, lazy popped up and said everything was fine with the installation of neorg. I put the line I removed back in it's place and still everything wokred. Wish I could provide more info on the issue. Now I can't recreate it. Thanks anyways!

**Reply by andreadev-it** [reply]
Ok, well I'm glad it's working now 🙂
Have a nice day!
